apps/server/doc/code-style.md

```diff
@@ -1,5 +1,33 @@
 # Code Style
 
+## Function
```

Comment:
Maybe you can also say some words of the total amount of parameters in a functions. No param perfect, one ok, 2 maybe, 3 if no other way, for more use objects, but keep in mind that any variation of this parameter inputs with this values must be added as test. (for 3 params minimum of 9 tests, mostly more if the values of the parameters include something like true/false of similar values)

Reply:
I think thats a general clean code coding rule, and not a project specific code style

Reply:
ok